### PR TITLE
databases/redis: add reset db button

### PR DESCRIPTION
**Pull request by mimugmail**
Add button to reset db.
My only problem is the scaling of the button, any idea how to put it on the correct place? :)

image

### CODE REVIEW

File: databases/redis/src/opnsense/mvc/app/views/OPNsense/Redis/index.volt [hunk 1]
```diff
@@ -87,4 +102,5 @@ $( document ).ready(function() {
 
 <div class="content-box tab-content">
     {{ partial("layout_partials/base_tabs_content",['formData':settings]) }}
+    <button class="btn pull-right" id="resetdbAct" type="button"><b>{{ lang._('Reset') }}</b> <i id="resetdbAct_progress" class=""></i></button>
```
Comment (fichtner) [inline databases/redis/src/opnsense/mvc/app/views/OPNsense/Redis/index.volt]:
```suggestion
    <button class="btn btn-default" id="resetdbAct" type="button"><b>{{ lang._('Reset') }}</b> <i id="resetdbAct_progress" class=""></i></button>
```

**Comment by mimugmail** [comment]
Now its this:

image

**Comment by fichtner** [comment]
I can fix this post-merge if you want. Not sure what's going on in the CSS.

**Comment by mimugmail** [comment]
Would be awesome! :)

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**